### PR TITLE
💄 Homepage - Navigation Bar Remodel 

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per #1806, we have 
- [ ] Changed the top navigation bar (non-sticky) so that the 'Star to GitHub' is moved to the middle 
- [ ] Added a TinaCloud to the NavBar which redirects to https://app.tina.io/signin 
- [ ] Added a Subscribe button into the navbar which opens a modal to allow users to subscribe with a first name, last name and email address
- [ ] Added a Book a demo button so that the main CTA is always visible on the sticky header 

_Note all of this has been test passed by Betty_

Screenshot 2024-05-31 at 5 59 04 pm

**Figure: New Nav Bar!**

Screenshot 2024-05-31 at 5 13 55 pm

**Laptop view of Modal**

Screenshot 2024-05-31 at 5 16 22 pm

**Tablet view of Modal**

Screenshot 2024-05-31 at 4 27 35 pm

**Phone view of Modal**

### CODE REVIEW

File: components/layout/Navbar.tsx [hunk 1]
```diff
@@ -140,28 +131,27 @@ export function Navbar({ }) {
...
             <LinkButton
-              link="https://app.tina.io/signin"
-              color="white"
-              size="small"
-            >
-              Log In
-            </LinkButton>
-            <LinkButton
-              link="https://app.tina.io/register"
+              link="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
```
Comment (bradystroud) [inline components/layout/Navbar.tsx:139]: what happened to `https://app.tina.io/register`?
Does it still work?

Would definitely be nicer if `https://app.tina.io/register` redirected to the ugly URL

File: components/layout/Navbar.tsx [hunk 2]
```diff
@@ -140,28 +131,27 @@ export function Navbar({ }) {
...
           ref={navRef}
-          className={`absolute ${stuck
-            ? `min-[1135px]:fixed shadow-sm bg-gradient-to-r from-[rgba(216,251,248,0.6)] to-[rgba(215,233,255,0.6)] backdrop-blur animate-slide-in top-0 p-4`
-            : `translate-y-2 px-4 pt-4 pb-6`
-            } z-40 w-full min-[1135px]:px-10 hidden min-[1135px]:flex items-center justify-between gap-6`}
+          className={`absolute ${
+            stuck
+              ? `min-[1135px]:fixed shadow-sm bg-gradient-to-r from-[rgba(216,251,248,0.6)] to-[rgba(215,233,255,0.6)] backdrop-blur animate-slide-in top-0 p-4`
+              : `translate-y-2 px-4 pt-4 pb-6`
+          } z-40 w-full min-[1135px]:px-10 hidden min-[1135px]:flex items-center justify-between gap-6`}
```
Comment (bradystroud) [inline components/layout/Navbar.tsx:154]: lots of arbitrary values in here - definitely try get these into tailwind config

File: components/forms/EmailForm.tsx [hunk 3]
```diff
@@ -1,182 +1,195 @@
...
+  const [isEntering, setIsEntering] = useState(false);
   const [isProcessing, setIsProcessing] = useState(false);
 
-  const [isSuccessOpen, toggleSuccess] = useToggle(false);
-  const [isErrorOpen, toggleError] = useToggle(false);
-  const [isDuplicateOpen, toggleDuplicate] = useToggle(false);
-
+  const [successMessage, setSuccessMessage] = useState('');
+  const [warningMessage, setWarningMessage] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
```
Comment (bradystroud) [inline components/forms/EmailForm.tsx:25]: Not a fan of having 3 different fields.

Could you do something like this instead?

```tsx
  const [message, setMessage] = useState({ text: '', type: '' });

  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
    e.preventDefault();
    setIsProcessing(true);
    setMessage({ text: '', type: '' });
    try {
      const result = await addToMailchimp(email, firstName, lastName);
      if (result.result === 'success') {
        setMessage({ text: "You've been added to the llama list.", type: 'success' });
      } else if (result.message === 'Bad Request') {
        setMessage({ text: "You're already in our herd!", type: 'warning' });
      } else {
        setMessage({ text: 'There was an error. Please try again.', type: 'error' });
      }
    } catch (error) {
      console.error('Error submitting email:', error);
      setMessage({ text: 'There was an error. Please try again.', type: 'error' });
    }
    setIsProcessing(false);
  };
```

**Review by bradystroud — CHANGES_REQUESTED** [review]
Hey @joshbermanssw 

Good start!
Let me know if you have questions about the feedback 🙂

File: components/layout/Navbar.tsx [hunk 4]
```diff
@@ -213,44 +203,51 @@ export function Navbar({ }) {
...
             <div className="flex items-center gap-6 min-[1135px]:gap-10">
               <div className="w-full flex justify-start items-center gap-4">
+                <Button color="white" size="small" onClick={openModal}>
+                  <MdEmail className="mr-2"/>
+                  Subscribe
+                </Button>
                 <LinkButton
-                  link="https://app.tina.io/signin"
-                  color="white"
+                  link="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
```
Comment (wicksipedia) [inline components/layout/Navbar.tsx:229]: can this _**not**_ be hardcoded

File: components/forms/EmailForm.tsx [hunk 5]
```diff
@@ -1,182 +1,195 @@
...
+    color: orange;
+    font-family: tuner-regular;
+    font-size: 14px;
+  }
+
+  .text-error {
+    color: #D22F2F;
+    font-family: tuner-regular;
+    font-size: 14px;
+  }
```
[the remark this answers is not in the document]
Comment (joshbermanssw) [reply components/forms/EmailForm.tsx:191]: ✅

File: components/layout/Navbar.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:139]: Two completely different links

File: components/forms/EmailForm.tsx (hunk 3 above)
Comment (joshbermanssw) [reply components/forms/EmailForm.tsx]: ✅ done

File: components/forms/EmailForm.tsx [hunk 6]
```diff
@@ -1,182 +1,195 @@
...
-            </div>
-          }
+    <div className="grid lg:grid-cols-2 grid-cols-1 gap-1 bg-white max-w-7xl mx-auto">
+      <div className="relative lg:h-full md:h-64 h-32">
+        <Image 
+          src={BettyWithLlama} 
+          alt="Betty with a llama" 
+          layout="fill"
+          objectFit="cover"
+          className="w-full h-full"
```
[the remark this answers is not in the document]
Comment (joshbermanssw) [reply components/forms/EmailForm.tsx:61]: the modal is not a CMS element

[the remark this answers is not in the document]
Comment (bradystroud) [reply components/forms/EmailForm.tsx:61]: Can you get the content from Tina?

[the remark this answers is not in the document]
Comment (joshbermanssw) [reply components/forms/EmailForm.tsx:61]: as discussed - this will be a different pbi

File: components/layout/Navbar.tsx (hunk 4 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:229]: as discussed - this will be a different pbi

File: components/layout/Navbar.tsx (hunk 2 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:154]: this was here prior - do you still want me to move it into tailwind config - the main thing is is the coloring but this style is used throughout

Comment (bradystroud) [reply components/layout/Navbar.tsx:154]: > this was here prior

Ok, make a tech debt PBI for this then 
https://www.ssw.com.au/rules/technical-debt/

File: components/forms/EmailForm.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (joshbermanssw) [reply components/forms/EmailForm.tsx:61]: https://github.com/tinacms/tina.io/issues/1822

File: components/layout/Navbar.tsx (hunk 4 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:229]: https://github.com/tinacms/tina.io/issues/1822

File: components/layout/Navbar.tsx (hunk 2 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:154]: https://github.com/tinacms/tina.io/issues/1823

**Review by bradystroud — APPROVED** [review]
LGTM thanks @joshbermanssw 🚀

I noticed a layout shift issue, but you can fix that in a separate PR

**Pull request merged**